packages/preview/lib/preview.js

```diff
@@ -124,6 +161,17 @@ const findVoidable = (pgdb) => {
   })
 }
 
+const findEmptyFollowup = (pgdb) => {
```

Comment:
returns a promise, so it should be async
```suggestion
const findEmptyFollowup = async (pgdb) => {
```

Reply:
Interesting. Thus far I was under the impression I'd have to declare a function `async` only if I do async/await or resolve/wait for Promises inside a function. Here I basically only "declare" a promise that someone else will have to resolve.

Let me check on that.

Reply:
There is no particular example available, but [this may be something along the lines](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/async_function#Simple_example). `findEmptyFollowup` returns an unresolved Promise on purpose.